extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/MySqlCdcIntegrationTest.java

```diff
@@ -49,7 +48,6 @@
 
 import static com.hazelcast.jet.Util.entry;
 
-@Ignore("https://github.com/hazelcast/hazelcast/issues/18451")
```

Comment:
Can we change the PR builder configuration somehow to make the test pass? Or break down the test into smaller units? Because I feel like we will reenable the test, it might continue failing, and people will have a hard time getting a green build.

Reply:
I don't know what failure you are talking about. The link in the issue is too old and doesn't work. I re-added the test to actually see the failure and be able to fix it. Must be some setup issue, because works just fine locally.

Reply:
I will fix it, not leave it like that. Once I see what needs fixing.

Reply:
Will try to dig out the logs from S3.

Reply:
Logs from all (well, except for a very small few) builds are stored in s3. Even though the jenkins run might be gone, all you need is the build name and build number. You can then download the logs from the web UI (https://console.aws.amazon.com/s3/buckets/j-artifacts/?region=us-east-1&tab=overview) or use the AWS CLI (see the last point [here](https://hazelcast.atlassian.net/wiki/spaces/~matko/pages/3106963520/Test+failure+hunting+tips))

Reply:
Yes, I obtained the logs in the end, but there's no helpful information in them, because they don't contain the logs for the extension modules. More on that in a sec.

Reply:
Ok, the log problem is being worked on. In the meantime, pls. merge this in, I need to see a failure with logs, can't fix it otherwise, because I don't know what the problem is.

Reply:
Sure.